pyproject.toml

```diff
@@ -1,33 +1,32 @@
 [tool.poetry]
 # required ones
-name = "geoips"         # geoips package available at the root of the project
+name = "geoips"                                          # geoips package available at the root of the project
```

Comment:
What linting are you using for TOML?  We should probably set that up in general, or we're going to end up with a lot of reformatting changes like this...

Reply:
How can I check that? I'm not sure.

Reply:
It is probably within your VSCode settings.  @jsolbrig  ?

Reply:
I think it's probably "Even Better TOML".